### PR TITLE
Feature/members

### PR DESCRIPTION
**Pull request by Osvaldoav**
### What does this PR do?

- Add Values section to Members
- Add SearchBar to Members
- Refactor Members and MembersGrid js files

#### Related issues

#78

### CODE REVIEW

File: src/components/Members/MembersGrid/MembersGrid.js [hunk 1]
```diff
@@ -32,315 +32,239 @@ type Member = {
...
-  /**
-  * Operations to hide and change modals on pressed arrow keys.
-  * @param {KeyboardEvent} event: React event
-  */
-  keyFunction = (event: KeyboardEvent) => {
-    if (event.keyCode === 27) {
-      this.handleHideModal()
+    return () => {
+      window.removeEventListener('resize', updateGridCols)
+      document.removeEventListener('keydown', keyListener)
```
Comment (AuroTB) [inline src/components/Members/MembersGrid/MembersGrid.js:154]: nice

**Review by AuroTB — CHANGES_REQUESTED** [review]
Excellent work! I really liked your PR. Impeccable code.

## Some UI/UX comments

**Values**
- Make upper and bottom padding smaller in the "Values" title
- Make text smaller in large view in the values description, I recommend to use vh (because its vertical)
- Make it fit the page (large and small view)
- make images fit the container, in large view it looks weird when repeting it

image
image
image

**Search bar**
- Map "software development" with "programmer", I think people will search for that
- Bug in galaxy fold dimensions, small view
image


**Join us button**
- Modify the button border color to #00FFFA
- Center in small view
image
image

**Comment by Osvaldoav** [comment]
> * Map "software development" with "programmer", I think people will search for that

I'd recommend to address this in the members json file; adding a "programmer" tag for every "software development" tag instead of hardcoding the match in the search engine.

**Comment by Osvaldoav** [comment]
All comments have been addressed. Please check this out @AuroTB  :)

**Review by AuroTB — APPROVED** [review]
LGTM

**Pull request merged**